src/containers/guest-home-page/login-form/LoginForm.jsx

```diff
@@ -70,7 +70,16 @@ const LoginForm = ({
         {t('login.forgotPassword')}
       </Typography>
 
-      <AppButton loading={authLoading} sx={styles.loginButton} type='submit'>
+      <AppButton
+        disabled={
+          !Object.values(data).every((value) => value) ||
+          !(data.password.length >= 8 && data.password.length <= 25) ||
+          !Object.values(errors).every((value) => !value)
```

Comment:
Maybe we should use validation rules from `src/utils/validations/common.ts`?

Reply:
Yes, right!
Changes are made,
Thank you for the comment